sim/team-validator.js

```diff
@@ -199,6 +199,14 @@ class Validator {
 		item = dex.getItem(set.item);
 		ability = dex.getAbility(set.ability);
 
+		let bannedAbilities = format.bannedAbilities || dex.getFormat('gen7almostanyability').bannedAbilities;
+		if (ruleTable.has('ignoreillegalabilities') && bannedAbilities.includes(set.ability)) {
```

Comment:
This essentially hard-codes AAA’s ability banlist into `’ignoreillegalabilities’`, something I intentionally didn’t do when I refactored it into its own ruleset because it made more sense for custom tournaments to be forced to ban these abilities outright than be forced to use AAA’s ability banlist as a baseline and still have to ban other abilities outright anyway.